homeassistant/components/sma/translations/en.json

```diff
@@ -0,0 +1,47 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Device is already configured",
+            "already_in_progress": "Configuration flow is already in progress"
+        },
+        "error": {
+            "cannot_connect": "Failed to connect",
+            "cannot_retrieve_device_info": "Successfully connected, but unable to retrieve the device information",
+            "invalid_auth": "Invalid authentication",
+            "unknown": "Unexpected error"
+        },
+        "step": {
+            "custom_sensor": {
```

Comment:
Unused now?

On a side now, we only collect the most minimal amount of data to get the integration setup and customizations should happen in the options flow.

Reply:
Fixed with ec0b8f7